src/assets/i18n/en.json

```diff
@@ -84,6 +85,7 @@
   "login.create.assignment.description": "You need to create a profile before you can create a job",
   "login.apply.assignment.title": "Search jobs",
   "login.apply.assignment.description": "You need to create a profile before you can search for a job",
+  "start.or": "or",
```

Comment:
Cool! Missed that, I'll take a look at it when I arrive today, and then update the job-details page as well, thanks!